mlflow/projects/utils.py

```diff
@@ -210,6 +210,9 @@ def _fetch_git_repo(uri, version, dst_dir):
     if version is not None:
         try:
             origin.fetch(refspec=version, depth=GIT_FETCH_DEPTH)
```

Comment:
```suggestion
            origin.fetch(refspec=version, depth=GIT_FETCH_DEPTH, tags=True)
```

Does this work?

Reply:
yes, it works

Reply:
@harupy checked it on my local and made the change, can you review it?